src/main/java/org/icgc/argo/program_service/services/ProgramService.java

```diff
@@ -283,6 +286,7 @@ private List<InstitutionEntity> filterAndAddInstitutions(@NonNull List<String> n
   public ProgramEntity updateProgram(
       @NonNull ProgramEntity programToUpdate,
       @NonNull ProgramEntity updatingProgram,
+      @NotNull DataCenterDetailsDTO dataCenterDetailsDTO,

```

Comment:
Confusing why annotating dataCenterDetailsDTO as NotNull, and in the facade we're still passing this value as null.

Reply:
Also notice this param is using `NotNull` and other params are using 'NonNull' from Lombok.

Reply:
I'd suggest removing this annotation if you really want to pass a null dataCenter value, as I see you have already an If condition to check null dataCenterDetailsDTO